### PR TITLE
bug: implimented blog hide functionality

### PR DESCRIPTION
**Pull request by voidhrithik**
**What does this pull request do? Explain your changes. (required)**

This pull request is mainly centered around the blogs and sanity. previously the hide and show feature inside sanity to show the blog on the website was not working correctly. this pr is to resolve that issue. additionally, the ability to write code snippets on a blog post is added.

**Specific updates (required)**

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**
N/A

**Screenshots (optional)**

**Checklist**

- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/www/lib/client.tsx [hunk 1]
```diff
@@ -9,6 +9,9 @@ const options = {
   // token: process.env.SANITY_API_TOKEN,
 };
 
+if (process.env.NODE_ENV === "development")
+  options["token"] = process.env.SANITY_API_TOKEN;
```
Comment (clacladev) [inline packages/www/lib/client.tsx:13]: Let's please wrap it inside curly brackets. Just to keep it consistent with the rest of the codebase.

File: packages/www/pages/blog/[slug].tsx [hunk 2]
```diff
@@ -31,7 +32,19 @@ const serializers = {
...
+        <Code
+          className={""}
+          language={props.node.language}
+          value={props.node.code}>
+          {props.node.code}
+        </Code>
+      );
+      // return (
+      //   <code lang={props.node.language || "text"}>{props.node.code}</code>
+      // );
```
Comment (clacladev) [inline packages/www/pages/blog/[slug].tsx]: Commented code should not be committed.

**Review by clacladev — CHANGES_REQUESTED** [review]
Just two minor fixed.

@hrithikroboto please fill the PR template so we know what to review for. Otherwise swgee will have to give us this context via Discord. The PR is the right place.

Thanks for the help

File: packages/www/pages/blog/[slug].tsx (hunk 2 above)
Comment (voidhrithik) [reply packages/www/pages/blog/[slug].tsx]: updated this on the latest commit

File: packages/www/lib/client.tsx (hunk 1 above)
Comment (voidhrithik) [reply packages/www/lib/client.tsx:13]: updated this on the latest commit

**Review by clacladev — APPROVED** [review]
good to go

**Pull request merged**